hermes/src/main/java/in/raveesh/hermes/Util.java

```diff
@@ -0,0 +1,16 @@
+package in.raveesh.hermes;
+
+import android.annotation.SuppressLint;
+import android.util.Log;
+
+/**
+ * Created by sahil on 15/5/15.
+ */
+public class Util {
+    public static final String LOG_TAG = "in.raveesh.hermesexample";
+
+    @SuppressLint("LongLogTag")
+    public static void log(String message) {
```

Comment:
Yup, correct. I had first untracked the Utils file in git and made the commit but thought that the logging would become confusing. Would change it to regular logging.